cli/commands/play.go

```diff
@@ -135,7 +135,6 @@ var run = func(cmd *cobra.Command, args []string) {
 	snakes := buildSnakesFromOptions()
```

Comment:
Changes in this package are all just stripping out the special casing of Royale hazards since they can be pulled from the board state now.